Main/src/main/java/com/main/acad/controller/Controller.java

```diff
@@ -5,33 +5,43 @@
 import com.main.acad.dao.Dao;
 import com.main.acad.entity.Chapter;
 import com.main.acad.serializator.MyJsonSerializer;
+import com.main.acad.service.Service;
+import com.main.acad.service.ServiceImplement;
 
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
-import java.io.PrintWriter;
+import java.sql.SQLException;
 import java.util.List;
 
 public class Controller {
+    private Service serviceImplement = new ServiceImplement();
```

Comment:
bad naming: better something like StrategySimulationService, FastEatService, EnergySaveService, TemperedGlassService, UserSettingsService - so, that it takes some business meaning